CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing <!-- omit in toc -->
+
+- [Introduction](#introduction)
+- [Ground Rules](#ground-rules)
+- [Code Contributions](#code-contributions)
+  - [Your First Code Contribution](#your-first-code-contribution)
+  - [Getting started](#getting-started)
+  - [Pull Requests](#pull-requests)
+- [Style Guides](#style-guides)
+  - [Git Commit Messages](#git-commit-messages)
+  - [Coding style](#coding-style)
+    - [Style preferences not handled by the linter](#style-preferences-not-handled-by-the-linter)
+
+# Introduction
+
+Thank you for considering contributing! It's people like you that make the open source community so great! 😊
+
+The tips and guidelines below are intended to help both contributors and maintainers respect each other's time and effect change as quickly as possible.
+
+- **Bugs and Feature Requests**: Follow the templates [here](https://github.com/melink14/rikaikun/issues/new/choose) to report problems or ideas. Don't be surprised if you're asked for clarification or opinions on solutions.
+- **Code**: take a look at the open issues. Even if you can't write code, commenting on them, showing that you care about a given issue matters. It helps us triage them.
```

Comment:
This section talks more about issues than code. Should it be renamed?

```suggestion
- **Code**: Take a look at the open issues. Even if you are unable to implement a solution, commenting on issues with your opinions and experiences is greatly appreciated and helpful for triage.```

Reply:
maybe Implementation? Maybe Fixes?